1.segment_images/scripts/4.reconstruct_3D_masks.py

```diff
@@ -100,28 +112,50 @@
 
 
 image = tifffile.imread(input_image_dir)
-# image = (image - image.min()) / (image.max() - image.min()) * 255
-image = image.astype(np.uint8)
+if np.unique(image).max() > 255:
+    image = image.astype(np.uint16)
+else:
+    image = image.astype(np.uint8)
 
 
 # In[4]:
 
 
-skimage.measure.regionprops_table(image[1, :, :])
+cordinates = {
+    "original_label": [],
+    "slice": [],
+    "centroid-0": [],
+    "centroid-1": [],
+}
 
+for slice in range(image.shape[0]):
+    props = skimage.measure.regionprops_table(
+        image[slice, :, :], properties=["label", "centroid"]
+    )
 
-# In[ ]:
```

Comment:
Recommend confirming the notebook is run sequentially and also when looking at the plot with the centroid coordinates, I wasn't sure what masks where being used. Recommend including a title to the plot that explicitly states this, as it will be easier for someone reviewing the PR. I believe this notebook is showing organoids and those center x,y coords do not look to match the images below. I could be mistaken so I appreciate any clarification.